debian/pop-default-settings.postinst

```diff
@@ -86,7 +86,8 @@ case "$1" in
         # Add Pop proprietary repo if it is missing.
         RELEASE="$(lsb_release -cs)"
         REPO="deb http://apt.pop-os.org/proprietary ${RELEASE} main"
-        if ! grep "${REPO}" /etc/apt/sources.list > /dev/null
+        URI="http://apt.pop-os.org/proprietary"
```

Comment:
I would prefer this to be URL and for REPO to be constructed using it:

```
URL="http://apt.pop-os.org/proprietary"
REPO="deb ${URL} ${RELEASE} main"
```

Reply:
Done in https://github.com/pop-os/default-settings/pull/99/commits/772609987cd6cfacdc800b3a3c3ce54dfa118f0d.